docs/06-patterns/navigation.mdx

```diff
@@ -59,3 +59,23 @@ Breadcrumbs provide users with a critical indicator of where they are within the
 - Override breadcrumb behavior to be reflective of history.
 - Redirect breadcrumbs to compensate for a lack of a landing page.
 - Use title case.
+
+---
+
+## Return to previous
+
+Because of Grafana’s technical and information architecture, it is a common experience for a user to work on an item in one area and then follow a link to see the results of their work (such as SLO configuration that creates alert rules or cloud apps that create dashboards. Additionally, the more we weave cloud app experiences into core Grafana through extensions, the more we link our users from one branch of the information architecture “tree” to a very distant branch.
```

Comment:
```suggestion
Because of Grafana’s technical and information architecture, it is a common experience for a user to work on an item in one area and then follow a link to see the results of their work, ending in a different area (such as SLO configuration that creates alert rules or cloud apps that create dashboards). Additionally, the more we weave cloud app experiences into core Grafana through extensions, the more we link our users from one branch of the information architecture “tree” to a very distant branch.
```

Reply:
I missed the explanation about "the user starts in one area and then ends in a different one" but you might want to rephrase it. There was also the `)` missed.